src/SdssShape.cc

```diff
@@ -190,34 +190,23 @@ geom::Box2I computeAdaptiveMomentsBBox(geom::Box2I const &bbox,      // full ima
 /*
  * Calculate weighted moments of an object up to 2nd order
  */
-template <bool instFluxOnly, typename ImageT>
```

Comment:
Why is the option to compute `instFluxOnly` removed? It may not be used regularly, but it seems like a perfectly good option to have.

Reply:
calcmon is overloaded now instFluxOnly instantiated two different versions of the functions with C style pointers.One could rename calcmon to make this explicit.

Reply:
Ah, yes. Makes sense. I'm happy as long as there's an option to calculate only the `instFlux`.

Reply:
Yeah, the old code was C translated into bad C++.